doc/source/master_list.bib

```diff
@@ -47,6 +47,7 @@ @string{BAMS
 @string{BLM = {Boundary-Layer Meteorol.}}
 @string{AG = {Ann. Glaciol.}}
 @string{PHYS = {Physica D}}
+@string{JAMES = {J. Adv. Modeling the Earth Sys.}}
```

Comment:
It looks like Lettie's 2019 paper is JAMES too. Would you mind fixing hers like you did for Brady et al. 2019?